llvm/test/tools/llvm-dwarfdump/X86/prettyprint_type_units.s

```diff
@@ -1,403 +1,269 @@
+        .short  5                               # DWARF version number
+        .byte   1                               # DWARF Unit Type
+        .byte   8                               # Address Size (in bytes)
+        .long   .debug_abbrev                   # Offset Into Abbrev. Section
+        .byte   3                               # Abbrev [3] 0xc:0x5f DW_TAG_compile_unit
+        .byte   0                               # DW_AT_producer
+        .short  33                              # DW_AT_language
+        .byte   1                               # DW_AT_name
+        .long   .Lstr_offsets_base0             # DW_AT_str_offsets_base
+        .byte   2                               # DW_AT_comp_dir
+        .byte   4                               # Abbrev [4] DW_TAG_variable
+        .asciz  "s"                             # DW_AT_name
+        .long   .LS_decl-.Lcu_begin0            # DW_AT_type
+.LS_decl:
+        .byte   9                               # Abbrev [9] DW_TAG_structure_type
+        .quad   S_sig                           # DW_AT_signature
+        .byte   8                               # Abbrev [8] 0x54:0x4 DW_TAG_base_type
+        .byte   4                               # DW_AT_name
+        .byte   5                               # DW_AT_encoding
+        .byte   4                               # DW_AT_byte_size
+        .byte   0                               # End Of Children Mark
+.Ldebug_info_end2:
+        .section        .debug_info,"G",@progbits,S_sig,comdat
+.Ltu_begin2:
+        .long   .Ldebug_info_end3-.Ldebug_info_start3 # Length of Unit
+.Ldebug_info_start3:
+        .short  5                               # DWARF version number
+        .byte   2                               # DWARF Unit Type
+        .byte   8                               # Address Size (in bytes)
+        .long   .debug_abbrev                   # Offset Into Abbrev. Section
+        .quad   S_sig                           # Type Signature
+        .long   .LS_def-.Ltu_begin2             # Type DIE Offset
+        .byte   1                               # Abbrev [1] DW_TAG_type_unit
+        .short  33                              # DW_AT_language
+        .long   .Lstr_offsets_base0             # DW_AT_str_offsets_base
+.LS_def:
+        .byte   13                              # Abbrev [13] DW_TAG_structure_type
+        .asciz  "S"                             # DW_AT_name (simplified template name)
+        .byte   6                               # Abbrev [6] 0x46:0xd DW_TAG_GNU_template_parameter_pack
+        .byte   5                               # DW_AT_name
+        .byte   7                               # Abbrev [7] 0x48:0x5 DW_TAG_template_type_parameter
+        .long   .Lt1_decl-.Ltu_begin2           # DW_AT_type
+        .byte   7                               # Abbrev [7] 0x4d:0x5 DW_TAG_template_type_parameter
+        # Simulate DWARF emitted by GCC where the signature is directly in the type attribute.
```

Comment:
Is this comment in the right place? I'd expect it to be a line or two lower? (in the third template type parameter, that has `.quad t1_sig`?)

Reply:
You're correct. I guess it must have moved while I was refactoring things..